moveit_core/online_signal_smoothing/src/butterworth_filter.cpp

```diff
@@ -38,6 +38,7 @@
 
 #include <moveit/online_signal_smoothing/butterworth_filter.h>
 #include <rclcpp/clock.hpp>
+#include <rclcpp/rclcpp.hpp>
```

Comment:
I think this can be removed. `rclcpp.hpp` is an expensive header in terms of compilation time.

Reply:
Thanks ! forgot to remove that one after including it for some debugging.